R/data.table.R

```diff
@@ -12,9 +12,16 @@ methods::setPackageName("data.table",.global)
 #   (2) export() in NAMESPACE
 #   (3) add to vignettes/datatable-importing.Rmd#globals section
 .SD = .N = .I = .GRP = .NGRP = .BY = .EACHI = NULL
+J = function(...) {
+  stopf("The function 'J' should not be called here. 'J' is intended for use within the 'i' argument of data.table operations")
+}
+
+. = function(...) {
+  stopf("The function '.' should not be called here. '.' can be used as an alias for 'list' within the square brackets of a data.table, DT[.]")
+}
+
 # These are exported to prevent NOTEs from R CMD check, and checkUsage via compiler.
 # But also exporting them makes it clear (to users and other packages) that data.table uses these as symbols.
-# And NULL makes it clear (to the R's mask check on loading) that they're variables not functions.
```

Comment:
restore this, it still applies to `.SD = ... = .EACHI`.

I would group these functions next to `:=` instead:

https://github.com/Rdatatable/data.table/blob/9be687206c2d640a889d9dc2d6592b613645ca45/R/data.table.R#L2768-L2771

Reply:
done :)